cli/src/main/kotlin/commands/ScannerCommand.kt

```diff
@@ -190,6 +182,25 @@ object ScannerCommand : CommandWithHelp() {
 
         println("Using scanner '${scanner.scannerName}' with '${ScanResultsStorage.storage.javaClass.simpleName}'.")
 
+        val localFileStorageLogFunction: ((String) -> Unit)? = when {
```

Comment:
I think the function is quite large already and for readability it'd be better to
extract lines 185 - 201 into a separate function. E.g.: fun  logFilesCount

Reply:
I've moved out the whole scanner configuration in a separate commit.

Reply:
Thanks!